FirebaseAuth/Tests/SampleSwift/AuthenticationExample.xcodeproj/project.pbxproj

```diff
@@ -393,7 +393,7 @@
 				DE8B63712BEC2FB900607B82 /* GoogleSignIn */,
 				DE8B63732BEC2FB900607B82 /* GoogleSignInSwift */,
 				DE8B63762BEC302200607B82 /* FacebookLogin */,
-				DEE261C42C21E9F500EECAC5 /* RecaptchaEnterprise */,
+				B899DE642D071E6200D3A83D /* RecaptchaEnterprise */,
```

Comment:
<img width="1327" alt="image" src="https://github.com/user-attachments/assets/1cd89af4-1cf1-432c-9d67-a3b57f8ad1ed">
Looks like tests are failing as it is not happy with the dependency. I see RecaptchaEnterprise is already present, do we need to update it again?